### DISCUSSION TITLE
where do i place the lines for permissions

### DISCUSSION THREAD

**Discussion by CrazySeppe** [spatie/laravel-permission#2407] [Q&A]
in the following link you guys give an example of lines of code that u can you to give permissions but no where it says what file we have to fill this in, I also assume im the only one who doesnt know but ye.
https://spatie.be/docs/laravel-permission/v5/basic-usage/role-permissions

code:
$user->assignRole('writer');

// You can also assign multiple roles at once
$user->assignRole('writer', 'admin');
// or as an array
$user->assignRole(['writer', 'admin']);

--> end of code

is "user" is being used as the table from sql?

**Comment by parallels999** [comment]
[Eloquent: Getting Started - Laravel - The PHP Framework For Web Artisans](https://laravel.com/docs/10.x/eloquent)
>where it says what file we have to fill this in

where you need it, also read [UI options](https://github.com/spatie/laravel-permission/blob/main/docs/advanced-usage/ui-options.md) examples, or search implementation info on internet

**Reply by CrazySeppe** [reply]
well i want to use permissions in my dashboard.blade.php, but do they need to be written in there? GUI examples? im new to this sorry i dont really understand

**Reply by parallels999** [reply]
> dashboard.blade.php, but do they need to be written in there?

first you must learn basic programming concepts, read about Laravel and MVC

**Reply by CrazySeppe** [reply]
im familiar with the basics (php, css, html in laravel) but not with permission and not that skilled with database.

**Reply by parallels999** [reply]
> not with permission and not that skilled with database

your questions are not about the use of package, what you need to learn first is the basics in Laravel and MVC, if you read about it you will know how to use the code you asked

**Reply by CrazySeppe** [reply]
alr thx

**Reply by parallels999** [reply]
> read [UI options](https://github.com/spatie/laravel-permission/blob/main/docs/advanced-usage/ui-options.md) examples

Also that helps me a lot

**Comment by drbyte** [comment]
This demo may help as well:
https://github.com/drbyte/spatie-permissions-demo/pull/8/files
